src/main/java/com/nsnieto/keepinmind/controllers/AuthorizationController.java

```diff
@@ -1,51 +1,47 @@
 package com.nsnieto.keepinmind.controllers;
 
 
-
-import org.springframework.beans.factory.annotation.Autowired;
+import com.nsnieto.keepinmind.entities.User;
+import com.nsnieto.keepinmind.services.UserService;
 import org.springframework.http.ResponseEntity;
-import org.springframework.stereotype.Controller;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 
-import com.nsnieto.keepinmind.entities.User;
-import com.nsnieto.keepinmind.services.UserService;
-
+@RestController
```

Comment:
For reference: https://www.javacodegeeks.com/2017/08/difference-restcontroller-controller-annotation-spring-mvc-rest.html